packages/react-meteor-data/suspense/useSubscribe.ts

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react'
+import { EJSON } from 'meteor/ejson'
+import { Meteor } from 'meteor/meteor'
+import isEqual from 'lodash/isEqual'
+import { removeFromArray } from './useFind'
```

Comment:
This should be in a common `utils` files. Or even better, we could use `remove` from Lodash.